src/index.html

```diff
@@ -83,59 +86,107 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
           collaboration, and confidence.</h2>
       </section>
 
-      <section class="col-12 justify-content-center bg-grey">
-        <p id="description" class="text-center col-xs-8 col-md-6 col-lg-4 mx-auto my-5">
-          Girls Code Lincoln was established in January 2016 as Lincoln Girls Who Code. We host technology clubs
-          in Lincoln for 4th - 9th grade girls. Since inception we have held over 15 clubs and impacted over 200
-          girls.
-        </p>
+      <section id="about-us" class="col-12 justify-content-center bg-grey py-5">
+        <div class="col-xs-10 col-md-8 text-center mx-auto mb-5">
+          <h3 class="mb-3 font-weight-bold">
+            Girls Code Lincoln is a 501(c)(3) nonprofit organization. We strive to ignite passion for technology and leadership in young
+            girls with the long-term goal of closing the gender gap in S.T.E.M.
+          </h3>
+          <i>
+            We pride ourselves in partnering with Nebraska organizations to provide multiweek technology clubs free of charge.
```

Comment:
Use active voice, put what we do first. Maybe something like: "We host multiweek technology clubs for 4th--9th grade girls. All clubs are completely free and 100% volunteer-run. We partner with Nebraska organizations and build in-house curriculum to teach technology, leadership, and life skills."